src/meetings/components/meetingActionsBar/LeaveMeetingButton.test.tsx

```diff
@@ -25,13 +26,21 @@ beforeEach(() => {
 
 describe('LeaveMeetingButton', () => {
 	test('Default button status: closed', () => {
-		setup(<LeaveMeetingButton isHoovering />);
```

Comment:
I think you can remove `PipProvider` in these tests (and in `MeetingActionBar.tests.tsx` and `MeetingSkeleton.test.tsx`) because you added it in the `ProvidersWrapper`